frontend/src/app/components/View.js

```diff
@@ -92,12 +92,48 @@ export default class View extends React.Component {
     });
   }
 
+  renderUpgradeWarning() {
+    if (this.props.hasAddon === null) {
+      return null;
+    }
+    if (!!navigator.mozAddonManager) {
+      return null;
+    }
+    if (!this.props.isFirefox) {
+      return null;
+    }
+    let title = "Something is wrong!";
+    let copy = "Something has gone wrong with Test Pilot. Please file a bug at https://github.com/mozilla/testpilot/issues/new and mention this error message.";
+    if (!this.props.isMinFirefox) {
+      title = "Upgrade firefox";
+      copy = "The version of firefox which is installed is too old. Please upgrade.";
```

Comment:
Not sure how user-facing these error messages are, but these should probably be "Firefox" (same w/ "Upgrade firefox" above).